src/context/FoxEthProvider/FoxEthProvider.tsx

```diff
@@ -163,6 +164,9 @@ export const FoxEthProvider = ({ children }: FoxEthProviderProps) => {
   ).precision
   const featureFlags = useAppSelector(selectFeatureFlags)
 
+  // TODO: Remove this useEffect
+  // The reason why it is still here is because we use connectedWalletEthAddress both for the modals, and to display the DeFi cards/rows
+  // If we remove it now, we will be unable to get to the modal in the first place to change accounts
```

Comment:
We need this for now. To be tackled in https://github.com/shapeshift/web/issues/2615